articles/azure-resource-manager/managed-applications/create-uidefinition-functions.md

```diff
@@ -481,6 +481,48 @@ Assume `element1` and `element2` are undefined. The following example returns `"
 "[coalesce(steps('foo').element1, steps('foo').element2, 'foobar')]"
 ```
 
+   > [!NOTE]
```

Comment:
This section is probably a little long for a Note. Since it is really a description of how and when to use the function, the whole section would work better outside of a Note.

Reply:
I agree with your observation.